aboutme.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="./css/aboutme.css">
+    <title>togoville jazz festival aboutme</title>
+</head>
+<body>
+    <header>
+        <ul class="header-links">
+            <li><a href="#" class="social-link fa fa-twitter"></a></li>
+            <li><a href="#" class="social-link fa fa-facebook"></a></li>
+            <li><a class="lato-font-black" href="#">English</a></li>
+            <li><a class="lato-font-black" href="#">My page</a></li>
+            <li><a class="lato-font-black" href="#">Logout</a></li>
+        </ul>
+    </header>
+    <main>
+        <nav class="nav-bar">
+            <div class="hamburger-class">
+                <a href="#" ><img src="./img/iconmonstr-menu-1-240.png" alt=""></a>
```

Comment:
- [ ] To make our website accessible to everyone, it is a best practice to provide an `alt` attribute on all images. Kindly add the `alt` attribute to this one. ✍️